metricflow/test/integration/test_cases/itest_semi_additive_measure.yaml

```diff
@@ -0,0 +1,64 @@
+---
+integration_test:
+  name: semi_additive_measure_query_with_grouping
+  description: Tests selecting a measure with semi additive properties with a window_grouping
+  model: SIMPLE_MODEL
+  metrics: ["current_account_balance_by_user"]
+  group_bys: ["user"]
+  check_query: |
+    SELECT
+      a.user AS user
+      , SUM(a.current_account_balance_by_user) AS current_account_balance_by_user
+    FROM (
+      SELECT
+        ds
+        , user_id AS user
+        , account_balance AS current_account_balance_by_user
+      FROM (
```

Comment:
You can remove the `SELECT *` and just do `{{ source_schema }}.fct_accounts`. Same for the other cases where you do `SELECT *`.